src/main/java/com/angelbroking/smartapi/sample/Examples.java

```diff
@@ -1,382 +1,321 @@
+        // Order modify request will return order model which will contain only
+
+        OrderParams orderParams = new OrderParams();
+        orderParams.setVariety(Constants.VARIETY_STOPLOSS);
+        orderParams.setQuantity(324);
+        orderParams.setSymbolToken("1660");
+        orderParams.setExchange(Constants.EXCHANGE_NSE);
+        orderParams.setOrderType(Constants.ORDER_TYPE_STOPLOSS_LIMIT);
+        orderParams.setTradingSymbol("ITC-EQ");
+        orderParams.setProductType(Constants.PRODUCT_INTRADAY);
+        orderParams.setDuration(Constants.DURATION_DAY);
+        orderParams.setTransactionType(Constants.TRANSACTION_TYPE_BUY);
+        orderParams.setPrice(122.2);
+        orderParams.setTriggerPrice("209");
+        Order order = smartConnect.modifyOrder(orderInput.getOrderId(), orderParams, "STOPLOSS");
+
+        logger.debug("modifyOrder {}", order);
+        return order;
+
+    }
+
+    /**
+     * Cancel an order
+     *
+     * @return
+     */
+    public Order cancelOrder(SmartConnect smartConnect, Order modifyOrder) throws SmartAPIException {
+        // Order modify request will return order model which will contain only
+        // order_id.
+        // Cancel order will return order model which will only have orderId.
+        Order order = smartConnect.cancelOrder(modifyOrder.getOrderId(), Constants.VARIETY_STOPLOSS);
+        logger.debug("cancelOrder {}", order);
+        return order;
+    }
+
+    /**
+     * Get order details
+     */
+    public void getOrder(SmartConnect smartConnect) {
+        JSONObject orders = smartConnect.getOrderHistory();
+        logger.debug("getOrder {}", orders);
+    }
+
+    /**
+     * Get last price for multiple instruments at once. USers can either pass
+     * exchange with tradingsymbol or instrument token only. For example {NSE:NIFTY
+     * 50, BSE:SENSEX} or {256265, 265}
+     */
+    public void getLTP(SmartConnect smartConnect) {
+        String exchange = "NSE";
+        String symboltoken = "3045";
+        JSONObject ltpData = smartConnect.getLTP(exchange, Constants.SYMBOL_SBINEQ, symboltoken);
+        logger.debug("getLTP {}", ltpData);
+    }
+
+    /**
+     * Get tradebook
+     */
+    public void getTrades(SmartConnect smartConnect) throws SmartAPIException {
+        // Returns tradebook.
+        JSONObject trades = smartConnect.getTrades();
+
+        logger.debug("getTrades {}", trades);
+
+    }
+
+    /**
+     * Get RMS
+     */
+    public void getRMS(SmartConnect smartConnect) throws SmartAPIException {
+        // Returns RMS.
+        JSONObject response = smartConnect.getRMS();
+        logger.debug("getRMS {}", response);
+    }
+
+    /**
+     * Get Holdings
+     */
+    public void getHolding(SmartConnect smartConnect) throws SmartAPIException {
+        // Returns Holding.
+        JSONObject response = smartConnect.getHolding();
+        logger.debug("getHolding {}", response);

```

Comment:
As mentioned last time, please remove these debug statements. Please review the dentire code for these debug statements.

Reply:
have removed some debug statements and replaced some with log.info or log.error if necessary.